ca/certificates_test.go

```diff
@@ -402,7 +402,7 @@ func TestGetRemoteSignedCertificateWithPending(t *testing.T) {
 
 	completed := make(chan error)
 	go func() {
-		_, err := ca.GetRemoteSignedCertificate(context.Background(), csr, tc.WorkerToken, tc.RootCA.Pool, tc.Picker, nil, nil)
+		_, err := ca.GetRemoteSignedCertificate(context.Background(), csr, tc.WorkerToken, nil, tc.RootCA.Pool, tc.Picker, nil, nil)
```

Comment:
This is starting to look awful. There are 3 nils being called here. We might want to create some kind of struct to make this cleaner.